sugoi-api-event-webhook/src/main/java/fr/insee/sugoi/event/listener/webhook/SugoiEventWebHookProducer.java

```diff
@@ -51,22 +50,20 @@ public void handleContextStart(SugoiEvent cse) {
       case RESET_PASSWORD:
         User user = (User) cse.getProperties().get(EventKeysConfig.USER);
         String password = (String) cse.getProperties().get(EventKeysConfig.PASSWORD);
-        PasswordChangeRequest pcr =
-            (PasswordChangeRequest)
-                cse.getProperties().get(EventKeysConfig.PASSWORD_CHANGE_REQUEST);
         String realm = cse.getRealm();
         String userStorage = cse.getUserStorage() != null ? cse.getUserStorage() : "default";
+
         String webHookTag =
-            pcr.getProperties().get("tag") != null ? pcr.getProperties().get("tag") : "MAIL";
+            cse.getProperties().get(EventKeysConfig.WEBSERVICE_TAG) != null
```

Comment:
Maybe check that cse.getProperties().get(EventKeysConfig.WEBSERVICE_TAG)!="" too